_includes/_code/clickbox-a/main.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+// Uncomment lines 3 and 6 to view the visual layout at runtime.
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+// ClickboxA manages its own state.
+class ClickboxA extends StatefulWidget {
+  ClickboxA({Key key}) : super(key: key);
+
+  _ClickboxAState createState() => new _ClickboxAState();
+}
+
+class _ClickboxAState extends State<ClickboxA> {
```

Comment:
I'm worried that if we don't follow best practices in these examples, then developers will copy/paste this code and not learn the better pattern.